README.md

```diff
@@ -18,40 +18,99 @@ Some of these are tricky!  Don't trust your first instinct.
 
 **a)** `"1" == 1`
 
+True - Due to the == not looking for an exact similarity, but a 'truthiness'.
+Console says: True
```

Comment:
1a) You're right, but the reason why has to do with autoconvesion rather than truthiness. When comparing value, JavaScript will convert one type to another if the types don't match. With `==`, a string and a number being compared will be internally converted to strings (probably--maybe numbers) and then evaluated.